<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tests</class>
 <widget class="QDialog" name="Tests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="loginGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Login</string>
   </property>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="loginRadioButtonGroup">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="studentRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>105</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Student</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="contentManagerRadio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Content Manager</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="testCasesGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>461</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>      Test Case                            Description</string>
   </property>
   <widget class="QGroupBox" name="viewReqTextsGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="viewReqTextsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>View Required Texts</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewReqTextsButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="viewReqTextsDescription">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Request the list of required textbooks for Joe&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="viewTextDetailsGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="viewBookDetailsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>View Textbook Details</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="viewBookDetailsDescription">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Request the details for a textbook&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewBookDetailsButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="submitOrderGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="submitOrderLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit Order</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="submitOrderDescription">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Submit an order to the server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitOrderButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="addCourseGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="addCourseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Course</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="addCourseDescription">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Add a course to the cuTPS system&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addCourseButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="addBooksGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="addBookLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Books</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="addBookDescription">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Submit an order to the server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBookButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="passFailInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Result:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="resultInput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="passFailLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pass/Fail:</string>
   </property>
  </widget>
  <zorder>testCasesGroup</zorder>
  <zorder>loginGroup</zorder>
  <zorder>loginRadioButtonGroup</zorder>
  <zorder>line</zorder>
  <zorder>passFailInput</zorder>
  <zorder>resultLabel</zorder>
  <zorder>resultInput</zorder>
  <zorder>passFailLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
